src/app/modules/library/standard.ts

```diff
@@ -3,3 +3,9 @@ export class Standard {
   name: string = '';
   children: Standard[] = [];
 }
+
+export type ResearchProjectTypes = 'ARISE' | 'NLP-TIPS';
```

Comment:
I'm thinking we may want to make ResearchProjectTypes more generic and not restricted to specific values, similar to how we handle the standards now. That way, we don't need to change the code whenever we want to add a new research project option.

Reply:
Sounds good. Let's address in a future task.